cmd/rook-operator/restapi.go

```diff
@@ -65,11 +65,11 @@ func startAPI(cmd *cobra.Command, args []string) error {
 	context := clusterd.NewDaemonContext(cfg.dataDir, cfg.cephConfigOverride, cfg.logLevel)
 	factory := cephd.New()
 	apiCfg := &api.Config{
-		ConnFactory:    mon.NewConnectionFactoryWithClusterInfo(&cfg.clusterInfo),
-		CephFactory:    factory,
-		Port:           apiPort,
-		ClusterHandler: apik8s.New(clientset, context, &cfg.clusterInfo, factory, cfg.namespace, cfg.containerVersion),
-		InitRetryMs:    5000,
+		ConnFactory:            mon.NewConnectionFactoryWithClusterInfo(&cfg.clusterInfo),
+		CephFactory:            factory,
+		Port:                   apiPort,
+		ClusterHandler:         apik8s.New(clientset, context, &cfg.clusterInfo, factory, cfg.namespace, cfg.containerVersion),
+		RegisterMetricsRetryMs: 5000,
```

Comment:
Just wondering why this var needs to be an arg in the api Config? Do you think that is something users will configure?